Explorer/Assets/DCL/EmojiPanel/EmojiPanelView.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using DCL.UI;
+using UnityEngine;
+using UnityEngine.Serialization;
+using UnityEngine.UI;
+
+namespace DCL.Emoji
+{
+    public class EmojiPanelView : MonoBehaviour
+    {
+        public event Action<EmojiSectionName, bool> OnSectionSelected;
+        [SerializeField] public List<EmojiSectionToggle> emojiSections;
+
+        [SerializeField] public ScrollRect scrollView;
```

Comment:
Can you keep the properties style just like you with serialized fields in `EmojiSectionView` for example?